extras/create_snapshots_reports_scorecard.py

```diff
@@ -15,6 +15,7 @@
 import distutils.dir_util
 import glob
 import logging
+import numpy
```

Comment:
Please move this to the other non-stdlib imports.

Reply:
Good catch, please see https://github.com/cisagov/cyhy-reports/pull/116/commits/f8396987650c8a65565f1c14ae92f4828dd8a524.